src/client/components/loader-animation/loader-animation.style.css

```
@@ -0,0 +1,42 @@
+body {
+  background: #dbdfe2;
+}
+
+.loader {
+  position: absolute;
+  top: 47%;
+  left: 47%;
+}
+
+.bubble {
+  animation: expand 1s ease-in-out infinite;
```

Can we try with a little bit faster animation? 
```suggestion
  animation: expand .8s ease-in-out infinite;
```

---

yes we can do it .. I just liked it in easy and relaxed way .. hehe